README.md

```diff
@@ -25,10 +25,11 @@ Next, create a `WORKSPACE` file in your project root (or edit the existing one)
 containing:
 
 ```python
-git_repository(
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+http_archive(
     name = "build_bazel_rules_nodejs",
-    remote = "https://github.com/bazelbuild/rules_nodejs.git",
-    tag = "0.16.8",
+    urls = ["https://github.com/bazelbuild/rules_nodejs/releases/download/0.16.8/rules_nodejs-0.16.8.tar.gz"],
+    sha256 = "123",
```

Comment:
don't worry, as soon as this PR lands, I'll cut a release which will update the hash.
We don't have a tar.gz for 0.16.8 anyway.

Reply:
👍 I also like what rules_webtesting does. In their release notes they include the lines you need to add to your WORKSPACE file to use the version: https://github.com/bazelbuild/rules_webtesting/releases/tag/0.3.0

Reply:
yeah I'd like to automate the entire release, changelog, upload done by the script. next step?